src/lsdb/catalog/dataset/healpix_dataset.py

```diff
@@ -491,7 +492,24 @@ def dropna(
         to a single layer, multi-layer operations are not supported at this
         time.
         """
-        ndf = self._ddf.dropna(
-            axis=axis, how=how, thresh=thresh, on_nested=on_nested, subset=subset, ignore_index=ignore_index
-        )
+
+        def drop_na_part(df: npd.NestedFrame):
+            if df.index.name == SPATIAL_INDEX_COLUMN:
+                df = df.reset_index()
```

Comment:
Is the issue that dropna has unexpected behavior when working on a duplicated index? If so, what behavior? Does it result in an error? 

This may be a good thing to open an issue about, but I don't know if the behavior is localized to anything nested is doing versus pandas dropna behavior itself.

Reply:
I opened an issue on nested-dask https://github.com/lincc-frameworks/nested-dask/issues/60